### ISSUE TITLE
Add Lists

### ISSUE THREAD

**Issue by devopsmariocom** [wc-catalogue/blaze-elements#78]

**Comment by davidrus** [comment]
As we talk with @Hotell it does not make sense to create bl-list because we dont have css inheritance in web components, so it will be very difficult to create it. We cannot guarantee ordered list to work with browser user-agent default styles like image in attachment. At the end the blazecss library provide only unification styles for lists. For user it will be better to use native ul (ol) lists.

If you have any questions @elemementor @hopkins-tk  @elmariofredo, ask me or @Hotell directly.

![screen shot 2017-01-11 at 14 45 32](https://cloud.githubusercontent.com/assets/1128128/21850808/a3a7996c-d80c-11e6-9c1d-65a1e0e4c94b.png)

**Comment by davidrus** [comment]
I try some other options to develop it and my investigation was stopped by this resolution:
![screen shot 2017-01-17 at 13 57 26](https://cloud.githubusercontent.com/assets/1128128/22021507/113ce896-dcbe-11e6-8add-5a82b2f678cd.png)

There is still some issues which must be done before release:
- [ ] styling counter and their content must be merged
- [ ] align of number and text must be in one line
- [ ] un-ordered list mustn't have content with counter


What I tried:
- move counter into li (not-worked, item have counter from 0 for every line)
- add counter to :scoped css (not-worked)


If any of you have any other idea for this small issues, please let me know or checkout my branch and develop it :).

**Comment by davidrus** [comment]
Of course, link to the [branch is here](https://github.com/wc-catalogue/blaze-elements/tree/davidrus/lists).
